recipes-robot/systemd/robotserversystemd_1.00.bb

```diff
@@ -11,7 +11,10 @@ FILES_${PN} += "${systemd_system_unitdir}/opentrons.targets.wants"
 RDEPENDS_${PN} = "nginx python3-uvicorn"
 do_install_append() {
   install -d ${D}${systemd_system_unitdir}
+  install -d ${D}/etc/systemd/system
   install -m 0644 ${WORKDIR}/opentrons-robot-server.service ${D}${systemd_system_unitdir}
   install -m 0644 ${WORKDIR}/opentrons.target ${D}${systemd_system_unitdir}
   cp -r ${WORKDIR}/opentrons.targets.wants ${D}${systemd_system_unitdir}/opentrons.targets.wants
+  cd ${D}/etc/systemd/system
+  ln -s -r ${D}${systemd_system_unitdir}/opentrons.target default.target
```

Comment:
is this the right (lack of) `/` between `${D}` and ${systemd_system_unitdir}`?

Reply:
`export systemd_system_unitdir="/lib/systemd/system"` yocto's adding the leading slash for this one